src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -554,82 +554,23 @@ def get_non_host_dbs(dbs):
         else:
             return dbs[1:]
         
-
-    @staticmethod
-    def init_namespace_sync_d_interface_tables(dbs):
-        if_name_map = {}
-        if_alias_map = {}
-        if_id_map = {}
-        oid_sai_map = {}
-        oid_name_map = {}
-
-        """
-        all_ns_db - will have db_conn to all namespace DBs and
-        global db. First db in the list is global db.
-        Ignore first global db to get interface tables if there
-        are multiple namespaces.
-        """
-        Namespace.connect_namespace_dbs(dbs)
-        for db_conn in Namespace.get_non_host_dbs(dbs):
-            if_name_map_ns, \
-            if_alias_map_ns, \
-            if_id_map_ns, \
-            oid_sai_map_ns, \
-            oid_name_map_ns = init_sync_d_interface_tables(db_conn)
-            if_name_map.update(if_name_map_ns)
-            if_alias_map.update(if_alias_map_ns)
-            if_id_map.update(if_id_map_ns)
-            oid_sai_map.update(oid_sai_map_ns)
-            oid_name_map.update(oid_name_map_ns)
-
-        return if_name_map, if_alias_map, if_id_map, oid_sai_map, oid_name_map
-
     @staticmethod
-    def init_namespace_sync_d_lag_tables(dbs):
-
-        lag_name_if_name_map = {}
-        if_name_lag_name_map = {}
-        oid_lag_name_map = {}
-
-        """
-        all_ns_db - will have db_conn to all namespace DBs and
-        global db. First db in the list is global db.
-        Ignore first global db to get lag tables if
-        there are multiple namespaces.
-        """
-        Namespace.connect_namespace_dbs(dbs)
-        for db_conn in Namespace.get_non_host_dbs(dbs):
-            lag_name_if_name_map_ns, \
-            if_name_lag_name_map_ns, \
-            oid_lag_name_map_ns = init_sync_d_lag_tables(db_conn)
-            lag_name_if_name_map.update(lag_name_if_name_map_ns)
-            if_name_lag_name_map.update(if_name_lag_name_map_ns)
-            oid_lag_name_map.update(oid_lag_name_map_ns)
-
-        return lag_name_if_name_map, if_name_lag_name_map, oid_lag_name_map
-
-    @staticmethod
-    def init_namespace_sync_d_queue_tables(dbs):
-        port_queues_map = {}
-        queue_stat_map = {}
-        port_queue_list_map = {}
-
-        """
-        all_ns_db - will have db_conn to all namespace DBs and
-        global db. First db in the list is global db.
-        Ignore first global db to get queue tables if there
-        are multiple namespaces.
-        """
-        Namespace.connect_namespace_dbs(dbs)
+    def get_sync_d_from_all_namespace(per_namespace_func, dbs):
+        # return list of dictionaries retrieved from per namespace functions
```

Comment:
>return list of dictionaries [](start = 10, length = 27)

You did not return list of dictionaries

Reply:
we return list of dictionaries, the dictionaries will be the same as return from individual sync_d functions, just that they are merged together.

Reply:
Probably, we can say "return tuple of dictionaries"

Reply:
Updated as per review comment.